code/modules/mob/living/carbon/xenomorph/Abilities.dm

```diff
@@ -417,12 +417,9 @@
 		"<span class='xenowarning'>You begin to emit '[X.current_aura]' pheromones.</span>", null, 5)
 		playsound(X.loc, "alien_drool", 25)
 
-	if(X.hivenumber && X.hivenumber <= hive_datum.len)
-		var/datum/hive_status/hive = hive_datum[X.hivenumber]
-		if(isxenoqueen(X) && hive.xeno_leader_list.len && X.anchored)
```

Comment:
`X.anchored` is gone. Intended ?

Reply:
This would make it retain leader pheros when the queen is no longer in ovi.
I don't mind it as a change, but it doesn't seem intended.

Reply:
X.anchored is a hacky check, should be X.ovipositor

Reply:
Also the xeno proc to handle having leader pheros applied already checked for anchored (now ovipositor) and the idea is for all this stuff to be handled automatically without having to check the queen everywhere an update of leader pheros is triggered, see `/mob/living/carbon/Xenomorph/proc/handle_xeno_leader_pheromones(var/mob/living/carbon/Xenomorph/Queen/Q)` and `/datum/hive_status/proc/update_leader_pheromones()`